repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/core/tracing/rewritetransid/system-model.cfg.xml

```diff
@@ -13,4 +13,5 @@
                       default="true"/>
         </destinations>
     </repose-cluster>
+    <tracing-header rewrite-header="true"/>
```

Comment:
Even requiring the `enabled` attribute doesn't help when the element is not present.

Making the `tracing-header` element required seems like a good option, although it might require a change for some users as you point out.

As for the verb, do you mean inconsistent in that we currently use an attribute named `tracing-header`? If so, that's true, but since it's a major version anyway I'd prioritize making this configuration as clear as possible going forward.